isisd/isis_csm.c

```diff
@@ -147,8 +147,6 @@ isis_csm_state_change(int event, struct isis_circuit *circuit, void *arg)
 		assert(circuit);
 		switch (event) {
 		case ISIS_ENABLE:
-			zlog_warn("circuit already enabled");
-			break;
```

Comment:
The handling of the `IF_UP_FROM_Z` event causes the circuit to be marked as operational (`C_STATE_UP`). I'm not sure if treating the `ISIS_ENABLE` event the same as `IF_UP_FROM_Z` won't have consequences we're not aware of. That brings me to ask: is this change really necessary?

Reply:
Yes you are correct, we rectified it.